cumulusci/core/config.py

```diff
@@ -102,7 +104,7 @@ class BaseTaskFlowConfig(BaseConfig):
     def list_tasks(self):
         """ Returns a list of task info dictionaries with keys 'name' and 'description' """
         tasks = []
-        for task in self.tasks.keys():
+        for task in list(self.tasks.keys()):
```

Comment:
Might be nicer to use `iteritems()` instead of `list()`?

Reply:
iteritems is dead in py3